boards/arm/mimxrt1060_evk/board.cmake

```diff
@@ -4,5 +4,18 @@
 # SPDX-License-Identifier: Apache-2.0
 #
 
+set_ifndef(OPENSDA_FW jlink)
```

Comment:
At some point, can we discuss deprecating these sorts of variables? They are holdovers from the Kbuild days that are no longer needed.

You can `west flash -r jlink` or `west flash -r pyocd` to set the runner for a single invocation, and also just set BOARD_FLASH_RUNNER directly at cmake time to make the setting the default for an application build directory's lifetime.

Reply:
> Sure, let's discuss. I'm already using west this way and it's really nice. :) 

Awesome, glad to hear it, thanks :).

> Do we have any plans yet to update zephyr-app-commands to generate west commands instead of cmake/ninja commands?

Yup, it can do that already, with `:tool: west`, thanks to @carlescufi.